pipeline_dp/accumulator.py

```diff
@@ -3,7 +3,11 @@
 import pickle
 from dataclasses import dataclass
 from functools import reduce
+
+from typing import Iterable, Tuple
 import pipeline_dp
+from pipeline_dp.aggregate_params import NoiseKind
```

Comment:
Nit: import only modules (according to [styleguide](https://google.github.io/styleguide/pyguide.html#22-imports) )

Reply:
Done